polaris-icons/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 4.24.0
+
+- New icon `TemplateMinor` ([#6514](https://github.com/Shopify/polaris/pull/6514))
+- Updated metadata `TemplateMajor`
+
```

Comment:
```suggestion
```

If you run `yarn changeset` it will generate the change for you. Let me know if you want help with this. We don't add changes to this file.